src/main/kotlin/xta/game/creature/races/ElfRace.kt

```diff
@@ -0,0 +1,92 @@
+package xta.game.creature.races
+
+import xta.game.PlayerCharacter
+import xta.game.creature.Race
+import xta.game.creature.RacialStage
+import xta.game.creature.body.*
+import xta.game.stats.Stats
+
+object ElfRace : Race(32, "elf", 5) {
+    var elfHairColour = listOf("black", "leaf green", "golden blonde", "silver")
+    var elfSkinColour = listOf("dark", "light","tan")
+
+    object STAGE_ELF: RacialStage(this, "elf",
+        Stats.STR_MULT to -0.1,
+        Stats.TOU_MULT to -0.15,
+        Stats.SPE_MULT to +0.8,
+        Stats.INT_MULT to +0.8,
+        Stats.WIS_MULT to +0.6,
+        Stats.SENS to + 0.3
+    ){
+        override fun nameOf(creature: PlayerCharacter) = when {
```

Comment:
Not needed, see superclass implementation

Reply:
Actually you need to override if racial name should *not* have -taur suffix

Reply:
Okay, should have removed applicable cases, left wolfrace as is cause it checks for gender as well as a suffix if not taur.